swagger_server/controllers/survey_response.py

```diff
@@ -56,11 +56,21 @@ def add_survey_response(self, case_id, file):
             if collection_exercise:
                 exercise_ref = collection_exercise.get('exerciseRef')
                 survey_id = collection_exercise.get('surveyId')
+                ons_env.logger.debug('generating file name for upload with a survey ID of: {}'.format(survey_id))
+            else:
+                return self._invalid_upload()
+
+            # request survey service data from the API gateway
+            survey_service_data = self._get_survey_service(survey_id)
+
+            if survey_service_data:
+                survey_ref = survey_service_data('surveyRef')
+                ons_env.logger.debug('generating file name for survey reference id for upload with a survey ID of: {}'.format(survey_ref))
```

Comment:
I would expect this debug log message to be in the self._generate_file_name function